L'exécution de votre programme a déclenché une exception java.lang.ArrayIndexOutOfBoundsException : 12
 Merci de corriger votre code.
   at plm.universe.GridWorld.getCell (null:-1)
   at plm.universe.bugglequest.BuggleWorld.getCell (null:-1)
   at plm.runtime3.JediEscaper.hasLeftWall (plm.runtime3.JediEscaper:34)
   at plm.runtime3.JediEscaper.aMurOuest (plm.runtime3.JediEscaper:165)
   at plm.runtime3.JediEscaper.marqueCaseAutour (plm.runtime3.JediEscaper:95)
   at plm.runtime3.JediEscaper$$anonfun$run$1$$anonfun$apply$mcVI$sp$1.apply$mcVI$sp (plm.runtime3.JediEscaper:49)
   at plm.runtime3.JediEscaper$$anonfun$run$1$$anonfun$apply$mcVI$sp$1.apply (plm.runtime3.JediEscaper:47)
   at plm.runtime3.JediEscaper$$anonfun$run$1$$anonfun$apply$mcVI$sp$1.apply (plm.runtime3.JediEscaper:47)
   at scala.collection.immutable.Range.foreach (Range.scala:166)
   at plm.runtime3.JediEscaper$$anonfun$run$1.apply$mcVI$sp (plm.runtime3.JediEscaper:47)
   at plm.runtime3.JediEscaper$$anonfun$run$1.apply (plm.runtime3.JediEscaper:46)
   at plm.runtime3.JediEscaper$$anonfun$run$1.apply (plm.runtime3.JediEscaper:46)
   at scala.collection.immutable.Range.foreach (Range.scala:166)
   at plm.runtime3.JediEscaper.run (plm.runtime3.JediEscaper:46)
   at plm.core.lang.JVMCompiledLang.runEntity (null:-1)
   at plm.universe.World$1.run (null:-1)
   at java.lang.Thread.run (Thread.java:745)
